azext_edge/tests/edge/checks/conftest.py

```diff
@@ -141,12 +141,15 @@ def assert_dict_props(path: str, expected: str, obj: Dict[str, str]):
     val = obj
     for key in path.split("/"):
         val = val[key]
+
+    if val == expected:
+        assert val == expected
```

Comment:
this assert doesn't test for anything here

Reply:
true... let me revert it, it's a over refactoring

Reply:
no worries, I just reverted the change